parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetFileReader.java

```diff
@@ -730,6 +752,17 @@ public long getRecordCount() {
     return total;
   }
 
+  long getFilteredRecordCount() {
```

Comment:
@gszadovszky Could we make this public?

Reply:
I think, there is no problem making this public.
I would suggest creating a jira for the changes required for the spark integration (hopefully, not much). If everything works fine we'll try to do a rapid minor release (1.11.1) with these changes only.

Reply:
Thank you @gszadovszky. I have created https://issues.apache.org/jira/browse/PARQUET-1739